docs/multisigs/lido-on-x.md

```diff
@@ -0,0 +1,67 @@
+# Lido on X
+
+## 3.1 Lido on Polygon
+
+Address: [`0xd65Fa54F8DF43064dfd8dDF223A446fc638800A9`](https://app.safe.global/settings/setup?safe=0xd65Fa54F8DF43064dfd8dDF223A446fc638800A9)
+
+**Purpose of the multisig:** The multisig instance is used for signing off on upgrades and the introduction of parameter changes to the Lido on Polygon instance.
+
+**Quorum:** 3/5
+
+**Forum topic:** [Lido on Polygon V2 upgrade](https://research.lido.fi/t/lido-on-polygon-protocol-upgrade/3213/)
+**Snapshot:** [Lido on Polygon V2 upgrade](https://snapshot.org/#/lido-snapshot.eth/proposal/0x32f6f095dc4f7b97665b560781d6e6795da11f9c8218936a505575040038d70f)
+
+**Contracts and Roles:**
+
+stMatic token:[`0x9ee91F9f426fA633d227f7a9b000E28b9dfd8599`](https://etherscan.io/address/0x9ee91F9f426fA633d227f7a9b000E28b9dfd8599)
+- DAO_ROLE, 
+- PAUSE_ROLE, 
+- UNPAUSE_ROLE 
+
+Lido on Polygon NO Registry [`0x216B8b78e0632138dc38907dd089aAB601ED6EDC`](https://etherscan.io/address/0x216B8b78e0632138dc38907dd089aAB601ED6EDC)
+- DAO_ROLE, 
+- PAUSE_ROLE, 
+- UNPAUSE_ROLE
+
+**List of signers:**
+
+| Name | Address | Verification | Public verification |
+|: --- |: --- |: --- |: --- |
+| Jakov | 0x59d07dc34B135B17b87840a86BFF7302039E7EDf | https://etherscan.io/verifySig/11733 | https://twitter.com/defiyaco/status/1587222297936633857?s=61&t=zNY2z0koPK_WF8W9MSgaww |
+| vsh | 0x6E83d6f57012D74e0F131753f8B5Ab557824507D |  |  |
+| kadmil | 0x9A3f38AF97b791C85c043D46a64f56f87E0283D4 | https://etherscan.io/verifySig/17851 | https://twitter.com/kadmil_eth/status/1655865828544266242 |
+| Hamzah | 0xF20825cab04cc430Fe941eaa9C2Ae6b9DA9bc3B5 |  |  |
+| Aishwary | 0xc156C57231a9302D9f5C7b5eF22871cC25F40736 |  |  |
+
+## 3.2 Lido on Solana
```

Comment:
We might want to drop this section from the doc completely, because Lido on Solana is being sunset at the moment, and removing it now will save us a few clicks in the near future.

Reply:
agree